rest/src/routes.rs

```diff
@@ -92,38 +125,46 @@ impl<N: Network, B: BlockStorage<N>, P: ProgramStorage<N>> Server<N, B, P> {
             .and(with(self.ledger.clone()))
             .and_then(Self::get_validators);
 
-        // GET /testnet3/statePath/{commitment}
-        let get_state_path = warp::get()
+        // POST /testnet3/statePath/{commitment}
+        let get_state_path = warp::post()
```

Comment:
This should be a GET endpoint as per recent changes

Reply:
Yes you're right!

Reply:
Done!